website/src/categories/getting-started/developers.md

```diff
@@ -60,36 +77,34 @@ To get started using the SL Design System, you can install it via npm. Each comp
 
 ## Setup a theme
 
-To start using the SL Design System, you need to setup a theme for your application. The theme determines how your application looks. There is a theme for every major product within Sanoma Learning. You can install a theme as by installing the NPM package, in this example we use the `sanoma-learning` theme.
+To start using the SL Design System, you need to setup a theme for your application. The theme determines how your application looks. There is a theme for every major product within Sanoma Learning.
 
-<div class="ds-code">
+Please note that the webfonts of your application are not a part of the theme. There are different ways of loading the fonts an application used across the products in Sanoma Learning. If is up to you to load the webfonts in your application.
```

Comment:
There is a section about fonts a bit further down, maybe it is better to add this info there?

Reply:
The FAQ item is more about the fonts not working, even though you have loaded them. So I'll leave this bit here.